libs/libc/dirent/lib_nftw.c

```diff
@@ -198,7 +200,7 @@ do_nftw(FAR char *path, nftw_cb_t fn, int fdlimit, int flags, int level)
 
           if (strlen(de->d_name) > PATH_MAX - j)
             {
-              set_errno(ENAMETOOLONG);
+              _NX_SETERRNO(ENAMETOOLONG);
```

Comment:
seems like we need a similar change at line 113 and 164: `get_errno` -> `_NX_GETERRNO`